docs/install/openshift.md

```diff
@@ -28,89 +28,54 @@ Once the cluster is up and running, log in as the `cluster-admin` user.
 
 ### Namespace
 
-It is a good idea to create a new namespace for the operator.
+By default, the operator is installed into the `argocd-operator-system` namespace. To modify this, update the
+value of the `namespace` specified in the `config/default/kustomization.yaml` file. 
 
-```bash
-oc new-project argocd
-```
-
-The remaining resources will now be created in the new namespace.
-
-### RBAC
-
-Provision the ServiceAccounts, Roles and RoleBindings to set up RBAC for the operator.
-
-NOTE: The ClusterRoleBindings defined in `deploy/role_binding.yaml` use the `argocd` namespace. You will need to update these if using a different namespace.
-
-```bash
-oc create -f deploy/service_account.yaml
-oc create -f deploy/role.yaml
-oc create -f deploy/role_binding.yaml
-```
-
-Argo CD needs several ClusterRole resources to function, however the ClusterRoles have been refined to read-only for the cluster resources.
-
-To enable full cluster admin access on OpenShift, run the following command. This step is optional.
-
-``` bash
-oc adm policy add-cluster-role-to-user cluster-admin -z argocd-application-controller -n argocd
-```
-
-### CRDs
+### Deploy Operator
 
-Add the Argo CD CRDs to the cluster.
+Deploy the operator. This will create all the necessary resources, including the namespace. For running the make command you need to install go-lang package on your system.
 
 ```bash
-oc create -f deploy/argo-cd
+make deploy
 ```
 
-Add the Argo CD Operator CRDs to the cluster
+If you want to use your own custom operator container image, you can specify the image name using the `IMG` variable.
 
 ```bash
-oc create -f deploy/crds
+make deploy IMG=quay.io/my-org/argocd-operator:latest
 ```
 
-There should be four CRDs present for Argo CD on the cluster.
+The operator pod should start and enter a `Running` state after a few seconds.
 
 ```bash
-oc get crd | grep argo
+oc get pods -n argocd-operator-system
 ```
 
 ```bash
-NAME                       CREATED AT
-applications.argoproj.io   2019-11-09T06:36:59Z
-appprojects.argoproj.io    2019-11-09T06:36:59Z
-argocdexports.argoproj.io  2019-11-09T06:37:06Z
-argocds.argoproj.io        2019-11-09T06:37:06Z
-```
-
-### Deploy Operator
-
-Provision the operator using a Deployment manifest.
-
-```bash
-oc create -f deploy/operator.yaml
+NAME                                                  READY   STATUS    RESTARTS   AGE
+argocd-operator-controller-manager-6c449c6998-ts95w   2/2     Running   0          33s
 ```
+!!! info
+    If you see `Error: container's runAsUser breaks non-root policy`, means container wants to have admin privilege. run `oc adm policy add-scc-to-user privileged -z default -n argocd-operator-system` to enable admin on the namespace and change the following line in deployment resource: `runAsNonRoot: false`. This is a quick fix to make it running, this is not a suggested approach for *production*.
```

Comment:
what about this warning

Warning: would violate PodSecurity "restricted:v1.24": allowPrivilegeEscalation != false (container "kube-rbac-proxy" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (containers "kube-rbac-proxy", "manager" must set securityContext.capabilities.drop=["ALL"]), seccompProfile (pod or containers "kube-rbac-proxy", "manager" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")

Reply:
This will be fixed in the source code.